packages/waku/src/router/client.ts

```diff
@@ -498,11 +558,13 @@ const InnerRouter = ({
     });
   }, [initialRoute]);
 
+  const [, executeListeners] = routeChangeListenersRef.current;
```

Comment:
This should be more readable.
```suggestion
  const [routeChangeEvents, executeListeners] = routeChangeListenersRef.current;
```